bool near_exit
bool parking_card
